Maths/QuadraticFormula.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @function quadraticFormula
+ * @description This script will find the the roots of a quadratic equation.
+ * @param {number} a
+ * @param {number} b
+ * @param {number} c
+ * @return {string}
+ * @see https://en.wikipedia.org/wiki/Quadratic_formula
+ * @example quadraticFormula(1, -3, -4) = 'the roots are X1=4 and X2=-1'
+ * @example quadraticFormula(1, 5, 6) = 'the roots are X1=-2 and X2=-3'
+ * @example quadraticFormula(1, -3, 8) = 'the roots do not exist or the roots are imaginary'
+ */
+
+const quadraticFormula = (a, b, c) => {
+  if (typeof a !== 'number' || typeof b !== 'number' || typeof c !== 'number') {
+    return new TypeError('Some argument is not a number.')
+  }
+  const discriminant = (b ** 2) - (4 * a * c)
+  const denominator = 2 * a
+  let answer = ''
```

Comment:
Don't use strings; instead, return an array of solutions (empty array for no solutions, array with single item for one solution, two items for two solutions).

Reply:
Ok, I'll do that...thanks!